molecule_goss/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/verify.yml

```diff
@@ -42,21 +45,30 @@
         msg: "{{ test_files.files }}"
         verbosity: 3
 
-    - name: Copy Goss tests to remote  # noqa 208
+    - name: Copy defaults vars to goss vars
+      copy:
+        src: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/defaults/main.yml"
+        dest: "{{ goss_vars }}"
+      when: copy_defaults_vars
+
+    - name: Copy Goss tests to remote
       copy:
         src: "{{ item.path }}"
         dest: "{{ goss_test_directory }}/{{ item.path | basename }}"
+        mode: 0644
       with_items:
         - "{{ test_files.files }}"
 
     - name: Register test files
-      shell: "ls {{ goss_test_directory }}/test_*.yml"  # noqa 301
+      shell: "ls {{ goss_test_directory }}/test_*.yml"
+      changed_when: false
       register: test_files
 
     - name: Execute Goss tests
-      command: "{{ goss_bin }} -g {{ item }} validate --format {{ goss_format }}"  # noqa 301
+      command: "{{ goss_bin }} {% if copy_defaults_vars %}--vars {{ goss_vars }} {% endif %}-g {{ item }} validate --format {{ goss_format }}"
```

Comment:
please make use of  ": >" yaml wrapping to split this complex command to multiple lines.

Reply:
:+1: